docs/guides/theme_architecture/blocks/schema/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Schema
+parent: Blocks
+nav_order: 2
+---
+
+# Schema
+
+The block schema defines the [variables]({% link docs/guides/theme_architecture/blocks/schema/variables/index.md %}) a Creator is able to use to customise the content of a block.
+
```

Comment:
maybe a note about Creator friendly labels and hints

Reply:
or can do that in docs/guides/theme_architecture/blocks/schema/variables/index.md